src/Collection.php

```diff
@@ -253,7 +272,8 @@ public function reduce(callable $callable, $carry = null)
      * @param int|null $flag Collection::FILTER_USE_KEY or Collection::FILTER_USE_BOTH
      * @return static
      */
-    public function filter($callable = null, int $flag = null): self
+    // phpcs:ignore Squiz.WhiteSpace.ScopeKeywordSpacing.Incorrect -- Broken until 3.6.0
+    public function filter($callable = null, int $flag = null): static
```

Comment:
should this be `?int $flag = null`?

Reply:
Actually `?callable $callable = null, ?int $flag = null` by the looks of it 😄